packages/eslint-plugin-react-hooks/src/ExhaustiveDeps.js

```diff
@@ -67,6 +67,27 @@ export default {
       context.report(problem);
     }
 
+    /**
+     * SourceCode#getText that also works down to ESLint 3.0.0
+     */
+    function getSource(node) {
+      if (typeof context.getSource === 'function') {
+        return context.getSource(node);
+      } else {
+        return context.sourceCode.getText(node);
+      }
+    }
+    /**
+     * SourceCode#getScope that also works down to ESLint 3.0.0
+     */
+    function getScope(node) {
+      if (typeof context.getScope === 'function') {
+        return context.getScope();
+      } else {
+        return context.sourceCode.getScope(node);
+      }
+    }
+
     const scopeManager = context.getSourceCode().scopeManager;
```

Comment:
https://eslint.org/blog/2023/09/preparing-custom-rules-eslint-v9/#context-methods-becoming-properties
I noticed that this `context.getSourceCode()` was replaced with `context.sourceCode`, you might want to confirm that.
```js
const sourceCode = context.sourceCode ?? context.getSourceCode();
const scopeManager = sourceCode.scopeManager;
```

Reply:
`getSourceCode` was only deprecated.